modules/global-config/src/main/java/org/mule/runtime/globalconfig/internal/MavenConfigBuilder.java

```diff
@@ -55,6 +55,10 @@ public static MavenConfiguration buildMavenConfig(Config mavenConfig) {
           mavenConfig.hasPath("ignoreArtifactDescriptorRepositories")
               ? mavenConfig.getBoolean("ignoreArtifactDescriptorRepositories")
               : true;
+      boolean forcePolicyUpdateNever =
+          mavenConfig.hasPath("forcePolicyUpdateNever")
+              ? mavenConfig.getBoolean("forcePolicyUpdateNever")
+              : false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1125))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7725&issues=AWluT4_HotIXnmsUqKjR&open=AWluT4_HotIXnmsUqKjR)

[ISSUE_KEY]: AWluT4_HotIXnmsUqKjR